packages/eas-cli/src/analytics.ts

```diff
@@ -32,11 +34,33 @@ export async function initAsync(): Promise<void> {
   }
   const amplitudeEnabled = await UserSettings.getAsync('amplitudeEnabled', true);
```

Comment:
I think we will need to rename this to 'analyticsEnabled'. I left it out of this PR initially because it introduces unnecessary complexity when the goal for this PR is just to compare event volumes. I'm planning to introduce this change in the followup PR to remove amplitude.

Also, there is the scenario where this setting gets renamed and a user had previously opted out. In this case, the user will be opted back into enabling analytics. And I'm not quite sure how we want to handle that.

Reply:
We could write logic that renames the `amplitudeEnabled` to something else (e.g. `analyticsEnabled`) and remove this after few months (when the majority of users upgrades to a new EAS CLI).